erpnext/stock/report/stock_ageing/stock_ageing.py

```diff
@@ -15,6 +15,8 @@ def execute(filters=None):
 	to_date = filters["to_date"]
 	data = []
 	for item, item_dict in iteritems(item_details):
+
+		item_dict['fifo_queue'] = [item for item in item_dict if item[1]]
```

Comment:
```suggestion
		temp_fifo = [d for d in item_dict['fifo_queue'] if d[1]]
```
'item' variable name is already used above in for loop and you need to filter item_dict['fifo_queue'] not the entire item_dict
After filtering sort the temp_fifo

Reply:
added changes